.github/workflows/build.yml

```
@@ -16,7 +16,7 @@ jobs:
 
     strategy:
       matrix:
-        node-version: [12.x, 14.x, 16.x, 17.x]
+        node-version: [12.x, 14.x, 16.x, 18.x]
         webpack-version: [default, ^4.0.0]
         exclude:
           - node-version: 17.x
```

webpack 4 does not work on node 17+. Can you pleasechange the exclude here for node 18?
```suggestion
          - node-version: 18.x
```

---

done excluded node 18.